carma_wm/src/WMListenerWorker.cpp

```diff
@@ -137,6 +152,12 @@ void WMListenerWorker::roadwayObjectListCallback(const cav_msgs::RoadwayObstacle
 
 void WMListenerWorker::routeCallback(const cav_msgs::RouteConstPtr& route_msg)
 {
+
+  if(set_flag_==true)
```

Comment:
Flag name should be something more meaningful

Reply:
Done